### DISCUSSION TITLE
Prefer non-instrumental albums

### DISCUSSION THREAD

**Discussion by jasonpatrickellykrause** [RandomNinjaAtk/arr-scripts#159] [Ideas]
Lidarr  scripts are amazing, but every once in a while, they download the instrumental version of an album that is not listed as instrumental. Is there a way to prefer non-instrumental albums first?

**Comment by RandomNinjaAtk** [comment]
Your most likely seeing that happen because it's one of the releases for a particular release group on musicbrainz is a  instrumental release. There is no way to prevent it other than splitting that album into two release groups... Which can sometimes be a problem. Linkin Park was previously a good example where this was an issue but eventually was fixed by moving it to a separate release group....

If you can provide some examples. Maybe a modification could be made to avoid it, but it could have other consequences that wouldn't immediately be known...

**Reply by jasonpatrickellykrause** [reply]
Yeah, I can see how that would be an issue. 

https://musicbrainz.org/release-group/d9cb3910-77dc-4ac0-a6df-5959cb956c54

**Reply by RandomNinjaAtk** [reply]
@jasonpatrickellykrause 
Give the pushed changes a try....
https://github.com/RandomNinjaAtk/arr-scripts/commit/ebd5eb775797f6562dc177f65c331ac912133a37
https://github.com/RandomNinjaAtk/arr-scripts/commit/df1b9eb8468236424de853df9ea6376f16ad7a02

**Reply by jasonpatrickellykrause** [reply]
Just pulled in new changes, testing asap! I just added video downloads so this might be a moment.

**Reply by RandomNinjaAtk** [reply]
Tweaked it a bit more to improve it further: https://github.com/RandomNinjaAtk/arr-scripts/commit/1487dd7837bcf1ab77147a8a74ef1097cd00089a

**Reply by RandomNinjaAtk** [reply]
You'll want to update again, pushed a minor bug fix...

**Reply by jasonpatrickellykrause** [reply]
Initial test worked great, it picked up the correct album first try! 😎
